### ISSUE TITLE
windows build issue

### ISSUE THREAD

**Issue by GOBIKRISHNAN-R** [baresip/baresip#1689]
The fresh pull of baresip, re, rem in VS2022
Compile stops as

```
1>re-win32.lib(mem.obj) : error LNK2019: unresolved external symbol _btrace_println referenced in function _debug_handler
1>re-win32.lib(func.obj) : error LNK2019: unresolved external symbol _UINT32_C referenced in function _hash_fast
```

Any suggestions?

Thanks
Gobi

**Comment by GOBIKRISHNAN-R** [comment]
I added sdl.dll in my visual studio project file and It is compiling properly. But while running the baresip-win32.exe, I'm getting the **sdl.dll not found error**. The log is

```
baresip v1.1.0 Copyright (C) 2010 - 2021 Alfred E. Heggestad et al.
Local network addresses:
 {70519680-EDB2-42C9-A3C6-399CA21160BF}:  fe80::d5d:429f:609:e28d
 {70519680-EDB2-42C9-A3C6-399CA21160BF}:  169.254.226.141
 {084A1454-BD94-442C-8BC9-DA106314041E}:  fe80::99de:1d13:7e63:e505
 {084A1454-BD94-442C-8BC9-DA106314041E}:  169.254.229.5
 {97D3CCCF-0E1D-4E87-AC7B-5FF5E2BF3907}:  fe80::c419:ec8b:f097:e2a8
 {97D3CCCF-0E1D-4E87-AC7B-5FF5E2BF3907}:  169.254.226.168
 {905C2ED8-82DD-4520-8F8E-82C7712C5AE2}:  2402:3a80:133f:dede:48ea:20c9:32e3:f9c7
 {905C2ED8-82DD-4520-8F8E-82C7712C5AE2}:  2402:3a80:133f:dede:21a9:4498:890f:245
 {905C2ED8-82DD-4520-8F8E-82C7712C5AE2}:  fe80::48ea:20c9:32e3:f9c7
 {905C2ED8-82DD-4520-8F8E-82C7712C5AE2}:  192.168.8.27
 {B2E30189-18D4-4CF3-A1DB-817E783F0A0C}:  fe80::2800:2316:fafc:81
 {B2E30189-18D4-4CF3-A1DB-817E783F0A0C}:  169.254.0.129
aucodec: PCMU/8000/1
aucodec: PCMA/8000/1
winwave: output devices: 1, input devices: 1
ausrc: winwave
auplay: winwave
vidsrc: dshow
open: ./sdl.dll LoadLibraryA() failed
←[31mmodule sdl.dll: No such file or directory
←[;mmedianat: stun
medianat: turn
medianat: ice
Populated 0 accounts
account: No SIP accounts found
 -- check your config or add an account using 'uanew' command
Populated 3 contacts
Populated 2 audio codecs
Populated 0 audio filters
Populated 0 video codecs
Populated 0 video filters
baresip is ready.
```
And the error is

```
open: ./sdl.dll LoadLibraryA() failed
←[31mmodule sdl.dll: No such file or directory
←[;mmedianat: stun

```

Any suggestions, please?

**Comment by GOBIKRISHNAN-R** [comment]
Thanks... Fixed by referring to this issue:- #1197
